conduct.md

```diff
@@ -212,4 +214,8 @@ Any decision made by the instructor for the given infraction(s) will be final.
 ## Names of Individuals who Agree to Adhere to the Guide
 
 Janyl Jumadinova
-:heavy_exclamation_mark: TODO: Add your name
+Kevin Lee
+Adam Shinomiya
+Michael Ceccarelli
+Gabe Schwartz
+Liam Connelly
```

Comment:
Formatting the names like this will put them all on the same line once Markdown is rendered, It would be great to add commas between names to make this section more readable.